libc/vga/vga-init.greg.c

```diff
@@ -60,7 +60,7 @@ void _vga_reinit(struct Tty *tty, unsigned short starty, unsigned short startx,
             chr_ht, chr_wid, vid_buf, init_flags);
 }
 
-__attribute__((__constructor__)) static textstartup void _vga_init(void) {
```

Comment:
For the record, it's fine to use `__attribute__((__constructor__))` too :-) I started out with `textstartup` but I've been warming up more to the underscore syntax.

Reply:
Thank you @jart for the information!